taskcluster/docker/pushsnapscript/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# %ARG DOCKER_IMAGE_PARENT
+FROM $DOCKER_IMAGE_PARENT
+
+VOLUME /builds/worker/checkouts
+VOLUME /builds/worker/.cache
+
+# %include pushsnapscript/docker.d/image_setup.sh
+COPY topsrcdir/pushsnapscript/docker.d/image_setup.sh /usr/local/bin/
+RUN chmod a+x /usr/local/bin/image_setup.sh
```

Comment:
Nit: Why does this script need to have new permissions?

Reply:
It doesn't. I can remove this.